Tests/KotlinStandAloneTests/Resources/test.taskpaper

```diff
@@ -0,0 +1,17 @@
+test:
```

Comment:
Might be good if we can share this file and the util file across test targets. Not one for you to fix in this PR.